repository/repo_testing.go

```diff
@@ -156,7 +156,7 @@ func RepoTest(t *testing.T, creator RepoCreator, cleaner RepoCleaner) {
 
 		commits, err := repo.ListCommits("refs/bugs/ref2")
 		require.NoError(t, err)
-		assert.Equal(t, []Hash{commit1, commit2}, commits)
+		assert.ElementsMatch(t, []git.Hash{commit1, commit2}, commits)
```

Comment:
test is failing here, bad merge I suppose
```suggestion
		assert.ElementsMatch(t, []Hash{commit1, commit2}, commits)
```